runtime/altair/src/weights/pallet_treasury.rs

```diff
@@ -60,4 +60,8 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(RocksDbWeight::get().reads(1 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(1 as Weight))
 	}
+
+	fn spend() -> Weight {
```

Comment:
I guess we need to recompute the runtime weights in general right? We can do that in a following PR too